### PR TITLE
Setting up slider

### PR DESCRIPTION
**Pull request by syydneyyhardister**
Updated bundles.config.extensions.json, created a javascript, css, and html file for the slider component, and registered the slider component in amp-story-interactive.js

### CODE REVIEW

**Review by mszylkowski — APPROVED** [review]

File: extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (processprocess) [inline extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css:2]: 2021!

File: extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (processprocess) [inline extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js:2]: 2021!

**Review by processprocess — APPROVED** [review]
Little nits for the year but looks great!

File: extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css (hunk 1 above)
Comment (syydneyyhardister) [reply extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css]: Just changed the date

**Pull request merged**